low-level/files/opt/seagate/sspl/conf/sspl.conf.LR2.yaml

```diff
@@ -180,6 +180,22 @@ SYSTEMDWATCHDOG:
    monitor: true
    threaded: true
    monitored_services:
+      - hare-consul-agent.service
+      - hare-consul-agent-c1.service
+      - hare-consul-agent-c2.service
+      - elasticsearch.service
+      - kibana.service
+      - statsd.service
+      - rsyslog.service
+      - haproxy.service
+      - slapd.service
+      - corosync.service
+      - pacemaker.service
+      - pcsd.service
+      - lnet.service
+      - salt-master.service
+      - salt-minion.service
+      - glusterd.service
```

Comment:
Do we need monitor any 3rd party service for multipath? @ajaysrivas @madhavemuri ?

@shriya-deshmukh for enclosure inband accessibility, scsi-network-relay was one service in Cortx-v1, same applies still i believe, and should be here. Check exact name on hardware node & add here please.

Reply:
added scci-network-elay service.

Reply:
It would be good to check for "multipathd" service as well.

systemctl status multipathd

Reply:
thanks @madhavemuri.

@shriya-deshmukh please add 'multipathd' to the monitored service list